core/src/test/java/org/infinispan/eviction/impl/EvictionWithConcurrentOperationsTest.java

```diff
@@ -207,14 +207,13 @@ public void evicted(CacheEntriesEvictedEvent event) {
       Future<Object> get = fork(() -> cache.get(key1));
       readLatch.waitToBlock(30, TimeUnit.SECONDS);
 
+      // Ensures the eviction is complete of key1
+      put.get(30, TimeUnit.SECONDS);
```

Comment:
The javadoc says

> the read occurs after the eviction listener is notified (if passivation is enabled, it is written to disk) and after the entry is removed from the map

so I'm not sure why we need the reads to be concurrent with the write at all

Reply:
I believe it was an issue with leaking into other tests (not this one).  I don't remember off the top of my head.

Reply:
I believe it was an issue with leaking into other tests (not this one).  I don't remember off the top of my head.